case/Input/tube.in

```diff
@@ -2,19 +2,19 @@
 1.E-3                   ! eps_Ewld
 8                       ! PBspln_Ewld
 
-1                       ! nCellTypes
-1                       ! viscRat
-1.                      ! refRad
+3                       ! nCellTypes
+1 1 1                   ! viscRat
+1. 1. 1.                ! refRad
 .false.                 ! Deflate
 
 0.0     !
 0.0     !
 0.0     !
 
-10000                   ! Nt
-0.0014                  ! Ts
+10000                    ! Nt
+0.00014                  ! Ts
```

Comment:
this very small time step is an example of why this example `case` should be separate from a health rbc one, so we need more than one example

Reply:
Yes, will create new case-folder for rev2 for the changes.